extensions/classify/config.proto

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2020 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package istioclassify;
+
+// Configuration to classify traffic.
+message PluginConfig {
+  // next id: 2
+  // Multiple independent classification configurations.
+  repeated Classification classifications = 1;
```

Comment:
We still need to add documentation around the behavior of overlapping "output_attribute" i.e. what happens if the config has `operation` twice. Does the last one win?

Reply:
Classify is too narrow a use case for this implementation since this can be used in an abstract way.

Other name suggestion:
repeated AttributeConfig attributes = 1;

Reply:
`AttributeGeneration` would match the v1 name for these sorts of extensions. Not sure if that helps or not.

Reply:
I agree classification is too narrow, this is now an APA from v1.